auth.mkd

```diff
@@ -455,68 +431,68 @@ DRIP Link is important as its contents are used to provide trust in the DET/HI p
 
 This DRIP Authentication Message is used in conjunction with other DRIP SAM Types (such as the Manifest or the Wrapper) that contain data (e.g., the ASTM Location/Vector Message, Message Type 0x2) that is guaranteed to be unique, unpredictable, and easily cross-checked by the receiving device.
 
-A hash of the final link (`Broadcast Endorsement: HDA on UA`) in the Broadcast Endorsement chain SHOULD be included in each DRIP Manifest {{drip-manifest}}.
+A hash of the final link (`BE: HDA on UA`) in the Broadcast Endorsement chain SHOULD be included in each DRIP Manifest {{drip-manifest}}.
 
 ## DRIP Wrapper
 
 This SAM Type is used to wrap and sign over a list of other {{F3411}} Broadcast RID messages.
 
-The `evidence` section of the `Endorsement Structure for UA Signed Evidence` ({{bas}}) is populated with full (25-byte) {{F3411}} Broadcast RID messages. The ASTM Messages can be concatenated together to form a contiguous byte sequence, as shown in {{wrapper-fig}}.
-
-The maximum number of messages supported is 4. The messages MUST be in Message Type order as defined by {{F3411}}. All message types except Authentication (Message Type 0x2) and Message Pack (Message Type 0xF) are allowed.
-
-~~~~
- 0                   1                   2                   3
- 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
-+---------------+---------------+---------------+---------------+
-|                                                               |
-.                                                               .
-.                         ASTM Message(s)                       .
-.                                                               .
-|                                                               |
-+---------------+---------------+---------------+---------------+
-~~~~
-{:fig #wrapper-fig title="DRIP Wrapper Evidence"}
+The `evidence` section of the `Endorsement Structure for UA Signed Evidence` ({{bas}}) is populated with up to four ASTM {{F3411}} Messages in a contiguous octet sequence. Only ASTM Message Types 0x0, 0x1, 0x3, 0x4, and 0x5 are allowed and must be in Message Type order as defined by {{F3411}}. These messages MUST include the Message Type and Protocol Version octet and MUST NOT include the Message Counter octet (thus are fixed at 25 octets in length).
 
 ### Wrapped Count & Sanity Check
 
-When decoding a DRIP Wrapper on a receiver, a calculation of the number of messages wrapped and a sanity check can be performed by using the number of bytes (defined as `wrapperLength`) between the `UA DET` and the `VNB Timestamp by UA` as shown in {{wrapper-check}}.
+When decoding a DRIP Wrapper on a receiver, a calculation of the number of messages wrapped and a sanity check SHOULD be performed by using the number of octets (defined as `wrapperLength`) between the `VNA Timestamp by UA` and the `UA DET` as shown in {{wrapper-check}}.
```

Comment:
When this "SHOULD" can be safely ignored? If you that have any case, then consider MUST

Reply:
Made a MUST.